code/game/objects/effects/phased_mob.dm

```diff
@@ -10,7 +10,9 @@
 
 /obj/effect/dummy/phased_mob/Destroy()
 	// Eject contents if deleted somehow
-	var/area/destination_area = get_turf(src)
+	var/atom/drop_to = drop_location()
+	var/turf/just_in_case = get_turf(drop_to)
+	var/area/destination_area = just_in_case.loc
```

Comment:
```suggestion
	var/area/destination_area = get_area(drop_location())
```

Reply:
What if the destroyed mob is in nullspace, though? This will runtime then.
That possibility needs to be accounted for.
Runtiming on a `Destroy()` proc is very, very bad.

Reply:
I've added something to check for null returns from drop_location